plugins/@grouparoo/mailchimp/src/lib/export/exportProfile.ts

```diff
@@ -41,6 +42,11 @@ export const exportProfile: ExportProfilePluginMethod = async ({
     const oldMailchimpId = generateMailchimpId(
       oldProfileProperties["email_address"]
     );
+    if (!syncOperations.update) {
```

Comment:
It's important to know that even though oldEmail !== newEmail, it doesn't mean that they actually exist in the destination. At this point I don't think we know for sure that it's going to be an `update`. If oldEmail and newEmail don't exist, it would cause a creation instead.

Reply:
nice catch!